### ISSUE TITLE
How to get the test accuracy?

### ISSUE THREAD

**Issue by wyl0608** [blackeagle01/Abnormal_Event_Detection#7]
First of all, thanks for sharing, and I have some questions:
1. In test.py, I make threshold=0.00055,some normal events and some abnormal events were obtained, but the total number of events was only 24. The code in processor.py is to extract 1 frame every 5 seconds and I change to extract 5 frames every 1 second, so that about 2 seconds as a bunch (event) ,is it more reasonable ?
2. How do I get the test accuracy in this work, and how do you do it? Do I need to write a module for the reconstruction error of each frame to get the regularity score of the entire data set?
3. In processor.py, at line 76 "for video in videos[1:]:" seems to have lost a video.
Looking forward to your reply

**Comment by blackeagle01** [comment]
@wyl0608 haven't written a module for test accuracy. Will add it soon.
Also videos[1:] was to deal with an extra file that was somehow created in my OS. You can alter it!

**Comment by wyl0608** [comment]
thanks for the reply,waiting for the module.

**Comment by ataafarin** [comment]
I am also waiting for the test accuracy module,will it happen soon?

Thanks

**Comment by PBRAOS** [comment]
> First of all, thanks for sharing, and I have some questions:
> 
>     1. In test.py, I make threshold=0.00055,some normal events and some abnormal events were obtained, but the total number of events was only 24. The code in processor.py is to extract 1 frame every 5 seconds and I change to extract 5 frames every 1 second, so that about 2 seconds as a bunch (event) ,is it more reasonable ?
> 
>     2. How do I get the test accuracy in this work, and how do you do it? Do I need to write a module for the reconstruction error of each frame to get the regularity score of the entire data set?
> 
>     3. In processor.py, at line 76 "for video in videos[1:]:" seems to have lost a video.
>        Looking forward to your reply

Good point on the 1/5 vs 5 !. I am about to check if it makes a difference on the calibrations.

**Comment by AndyHon** [comment]
> 
> 
> thanks for the reply,waiting for the module.

How did you solve this problem?Can you share your code?
